libraries/render-utils/src/GeometryCache.cpp

```diff
@@ -150,7 +150,7 @@ void GeometryCache::computeSimpleHullPointListForShape(const int entityShape, co
 
 
         uniqueVerts.push_back(curVert);
-        outPointList.push_back(curVert * entityHalfExtents);
+        outPointList.push_back(curVert * entityExtents);
```

Comment:
Turns out we want to scale by the entity dimensions rather than halfDimensions.

Reply:
Thanks! This confirms an untested local change I had late Friday.  I wasn't sure about it though.